code/modules/hydroponics/grown/root.dm

```diff
@@ -28,10 +28,18 @@
 /obj/item/food/grown/carrot/attackby(obj/item/I, mob/user, params)
 	if(I.get_sharpness())
 		to_chat(user, span_notice("You sharpen the carrot into a shiv with [I]."))
-		var/obj/item/knife/shiv/carrot/Shiv = new /obj/item/knife/shiv/carrot
+		var/Cblade
```

Comment:
instead of using `Cblade`, give it a better var name and type. 
```suggestion
		var/obj/item/carrot_weapon
```

Reply:
Donity-done